README.md

```
@@ -1,155 +1,156 @@
-- [Kryo](https://github.com/EsotericSoftware/kryo) - Fast binary serialization framework
-- [Hibernate Validator](https://hibernate.org/validator/) - Bean validation framework
-- [Apache Commons Text](https://commons.apache.org/proper/commons-text/) - Text processing utilities
-
-This plugin is very much so still a WIP, please report any issues/requests to angrybadger#1 on discord.
+- **[Nexo](https://nexomc.com/)** — custom items in loot tables via direct API
+- **[PlaceholderAPI](https://placeholderapi.com/)** — stat placeholders
+- **[LumaGuilds](https://github.com/BadgersMC/LumaGuilds)** — guild-aware team behavior
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)